src/ui/Views/Organisation/RequestAccess.cshtml

```diff
@@ -1,14 +1,13 @@
 @model RequestAccessViewModel
 @{
-  Layout = "Shared/_OrganisationDetailsViewLayout";
   var fnv = ViewData.ModelState["FirstName"] != null && ViewData.ModelState["FirstName"].Errors.Any();
   var lnv = ViewData.ModelState["LastName"] != null && ViewData.ModelState["LastName"].Errors.Any();
   var eav = ViewData.ModelState["EmailAddress"] != null && ViewData.ModelState["EmailAddress"].Errors.Any();
   var ov = ViewData.ModelState["Organisation"] != null && ViewData.ModelState["Organisation"].Errors.Any();
   var rv = ViewData.ModelState["Reason"] != null && ViewData.ModelState["Reason"].Errors.Any();
 }
 
-<a href="@Url.Action("Show", "Organisation", new {ucasCode = @Model.TabViewModel.UcasCode})" class="govuk-back-link">Back</a>
+<a href="@Url.Action("Show", "Organisation", new {ucasCode = @ViewBag.UcasCode})" class="govuk-back-link">Back</a>
```

Comment:
Can you not use view model instead of view bag, as view bag is a touch more brittle.


Also it be nice if you keep the same name through out


prefer `InstitutionCode` over `UcasCode`

Then you can change it all to be short and snappy

`@Url.Action("Show", "Organisation", new {Model.InstitutionCode})`

Reply:
😨 There are 404 uses of the variable name ucasCode in the UI project, and 109 in the API project. This work is (a) not in scope of the card (b) pre-existing (c) an even larger refactor than the PR itself.

Please card it up instead @defong

Reply:
(needless to say that ucasCode may mean either instCode or courseCode, depending on context)